flixel/tweens/FlxTween.hx

```diff
@@ -825,4 +829,30 @@ class FlxTweenManager extends FlxBasic
 		while (_tweens.length > 0)
 			remove(_tweens[0]);
 	}
+	/**
+	 * Immediately updates all tweens of type PERSIST or ONESHOT through their endings, triggering their onComplete callbacks.
+	 * 
+	 * Note: if they haven't yet begun, this will first trigger their onStart callback.
+	 * 
+	 * Note: their onComplete callbacks are triggered in the next frame.  To trigger them immediately, call `FlxTween.manager.update(0);` after this function.
```

Comment:
That's just how tweens work, c.f. the the rest of the file, I can't remember exactly anymore